docs/Infrastructure.md

```diff
@@ -0,0 +1,35 @@
+# Suggested Infrastructure
+
+## Images
+
+There's three container images to deploy:
+
+- [Nginx server to host the frontend](../frontend/Dockerfile)
```

Comment:
Don't need this in staging/production, as there's an nginx-ingress controller that handles it. Good for dev, though :-)

Reply:
I see you have nginx hosting the SPA, so maybe it's needed if we deploy that way.